prov/util/src/util_attr.c

```diff
@@ -321,11 +500,15 @@ int fi_check_info(const struct fi_provider *prov,
 
 	if (user_info->caps & ~(prov_info->caps)) {
 		FI_INFO(prov, FI_LOG_CORE, "Unsupported capabilities\n");
+		FI_INFO(prov, FI_LOG_CORE, "Supported: %s\n", fi_tostr(&prov_info->caps, FI_TYPE_CAPS));
+		FI_INFO(prov, FI_LOG_CORE, "Requested: %s\n", fi_tostr(&user_info->caps, FI_TYPE_CAPS));
 		return -FI_ENODATA;
 	}
 
 	if ((user_info->mode & prov_info->mode) != prov_info->mode) {
 		FI_INFO(prov, FI_LOG_CORE, "needed mode not set\n");
+		FI_INFO(prov, FI_LOG_CORE, "Expected: %s\n", fi_tostr(&prov_info->mode, FI_TYPE_MODE));
+		FI_INFO(prov, FI_LOG_CORE, "Given: %s\n", fi_tostr(&user_info->mode, FI_TYPE_MODE));
```

Comment:
There are several places where the line lengths look a little too long.  Please keep closer to 80-90 characters.